risc0/r0vm/src/bin/r0vm.rs

```diff
@@ -40,6 +40,13 @@ struct Args {
     /// Add environment vairables in the form of NAME=value.
     #[clap(long, action = clap::ArgAction::Append)]
     env: Vec<String>,
+
+    /// Write "pprof" protobuf output of the guest's run to this file.
```

Comment:
It might be nice to show a end-to-end example set of commands in a `r0vm` crate level README ( and/or landing page for crates.io) of how to use profiling / other features of r0vm so its a little more accessible to users.

Reply:
Agreed, see: https://github.com/risc0/risc0/issues/472